sass/components/_toggle.scss

```diff
@@ -48,10 +48,10 @@ $toggle-default-font-size: 0.75rem;
 
     > .handle {
         position: absolute;
-        top: 0.4rem;
-        left: 0.4rem;
-        width: 2rem;
-        height: 2rem;
+        top: $size/7;
```

Comment:
Is it correct that these are supposed to be divided by 7? The 7 looks weird to me in particular because it's an odd number and because I thought our design system was using an 8 pixel grid

Reply:
dividing by 7 gives the same values are before though, so the 'lg' size looks the same.